src/singletons/Settings.hpp

```diff
@@ -84,6 +84,7 @@ class Settings : public ABSettings, public ConcurrentSettings
     BoolSetting showEmptyInput = {"/appearance/showEmptyInputBox", true};
     BoolSetting showMessageLength = {"/appearance/messages/showMessageLength",
                                      false};
+    IntSetting messageOverflow = {"/appearance/messages/messageOverflow", 0};
```

Comment:
Following the suggestion below, change type of this setting to `EnumSetting<T>` instead. Following that, change default value to, e.g. `FooBar::Allow`.